src/main/java/frc/robot/shooter/SpinUpCommand.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+package frc.robot.shooter;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class SpinUpCommand extends Command {
+    private final ShooterSubsystem subsystem;
+    private final double targetSpeed;
+
+    public SpinUpCommand(ShooterSubsystem subsystem, double targetSpeed) {
```

Comment:
I think we are going to standardize on not writing `Command` in every command class name. Change to `SpinUp`

Reply:
renamed